ocaml/xapi-cli-server/records.ml

```diff
@@ -5240,5 +5240,15 @@ let vtpm_record rpc session_id vtpm =
       ; make_field ~name:"is_protected"
           ~get:(fun () -> string_of_bool (x ()).API.vTPM_is_protected)
           ()
+      ; make_field ~name:"allowed-operations"
+          ~get:(fun () ->
+            map_and_concat Record_util.vtpm_operation_to_string
+              (x ()).API.vTPM_allowed_operations
+          )
+          ~get_set:(fun () ->
+            List.map Record_util.vtpm_operation_to_string
+              (x ()).API.vTPM_allowed_operations
```

Comment:
won't adding this method allow the user to set the allowed operations?

Reply:
I don't know; I notice that this is in other records. I was expecting to see only `get`